Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/.template.config/template.json

```diff
@@ -51,7 +51,11 @@
         },
         "NetCore": {
             "type": "computed",
-            "value": "(Framework == \"net9.0\" || Framework == \"net8.0\"|| Framework == \"net7.0\" || Framework == \"net6.0\")"
+            "value": "(Framework == \"net9.0\" || Framework == \"net8.0\" || Framework == \"net7.0\" || Framework == \"net6.0\")"
+        },
+        "FrameworkShouldUseV1Sdk": {
```

Comment:
Nit... it's not really the `V1Sdk` as it's a range of packages right? Should it just be `FrameworkShouldUseV1Worker` or something? Or just `...ShouldUseV1`? 

Not a big deal either way... just jumped out.